app/src/main/res/layout/fragment_about.xml

```diff
@@ -3,27 +3,27 @@
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 <ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
-    android:layout_width="fill_parent"
-    android:layout_height="fill_parent">
+            xmlns:app="http://schemas.android.com/apk/res-auto"
+            xmlns:tools="http://schemas.android.com/tools"
+            android:layout_width="fill_parent"
```

Comment:
Nit: fill_parent is deprecated by match_parent. I know you only changed the indent, but we should avoid ancient, deprecated attributes.

Reply:
I'm surprised `lint` doesn't point that out.